README.md

```diff
@@ -41,10 +41,16 @@ This is for development purposes only and if you are interested in running a min
 
 ## Installing Miner from Source
 
-First, you'll need [git](https://git-scm.com/). If you don't have it installed:
+### AMD64 (AMD/Intel)
 
+You need a processor with `avx` extensions. You can check this exists by running the following- if it is empty, your processor doesn't support AVX.
 ```bash
-sudo apt-get install git
+grep avx /proc/cpuinfo
+```
+Next, install [git](https://git-scm.com/) and build dependencies.
+
+```bash
+apt install -y git erlang libdbus-1-dev autoconf automake libtool flex libgmp-dev cmake libsodium-dev libssl-dev bison libsnappy-dev libclang-dev doxygen vim build-essential cargo parallel
```

Comment:
vim? doxygen? isn't bison in build-essentials?  can we declutter this some?

Reply:
Some of that was cargoculted, generally I added packages but didn't remove previous ones. But bison is NOT in build-essential. There might be some others that overlap, but I tested a couple dozen iterations and could only reduce it by a few.

The start of the list (which included doxygen) was [already in the README](https://github.com/helium/miner/commit/5e757014d5b51a00ba1de336e7c7d82961d7ad04#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R66). This just duplicates it to create full instructions for both amd and arm.

Using the ubuntu:20.04 docker image for testing, here's the minimum required list that I've been able to derive:

```
git erlang libdbus-1-dev autoconf automake libtool flex libgmp-dev cmake libsodium-dev libssl-dev bison build-essential cargo parallel 
```

It removes libsnappy-dev, libclang-dev, doxygen, and vim (of course).

my full test command is this:
```
apt update && DEBIAN_FRONTEND=noninteractive apt install -y git erlang libdbus-1-dev autoconf automake libtool flex libgmp-dev cmake libsodium-dev libssl-dev bison build-essential cargo parallel && git clone https://github.com/helium/miner && cd miner && make
```

I can update the commit if you want.